public/javascripts/index.js

```diff
@@ -1,19 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-const ButtonChoices = React.createClass({
-    render() {
-	return (
-	    <div id="buttons">
-	    <div id="button-group">
-	    <div id="capital-button">A</div>
-	    <div id="lowercase-button">a</div>
-	    </div>
-	    </div>
-	)
```

Comment:
Thanks for the feedback, Kris. My editor gets a little confused I think when mixing this JS and CSS so doesn't actually let me nest these divs any further. That's on a to do list though!